egs/chime4/s5_2ch/run.sh

```diff
@@ -7,11 +7,22 @@
 #                Mitsubishi Electric Research Labs (Shinji Watanabe)
 #  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
 
+# Added advanced baseline
+# Szu-Jui Chen 10/19/2017
+
 . ./path.sh
 . ./cmd.sh
+#####Baseline settings#####
+# Usage: 
+# For using original baseline, set baseline to chime4_official. In this case,
+# you don't need to run flatstart since we already provided the trained model in 
+# tools/ASR_models.
+# For advanced baseline, please run flatstart first to get the models.
+# By default, we assume you already got the models.
```

Comment:
can you please be more explicit here about what the user is supposed to run, e.g. say what they should run first and what they should run next.

Reply:
Sure. I will rewrite this part.